Assets/Scripts/BuddyManager.cs

```diff
@@ -23,6 +23,6 @@ public static void DecrementAllBuddyResources()
 			buddyStat.DecrementResources();
 		}
 
-		instance._buddyStats.RemoveAll( buddyStat => buddyStat.isAlive == false ); // Predicate statement
+		instance._buddyStats.RemoveAll( buddyStat => buddyStat.isAlive == false );
```

Comment:
That's a language feature, though. We shouldn't be putting comments in every time someone doesn't understand some aspect of the language. If you see it again and forget what it does, you can look up `RemoveAll()` and see that it takes a predicate statement as an argument.